WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/AppFileExplorer.cs

```diff
@@ -311,6 +311,11 @@ public class FileOrFolderInformation
             public long SizeInBytes { get; private set; }
 
             /// <summary>
+            /// Gets whether the current item is a folder
+            /// </summary>
+            public bool IsFolder { get { return this.Type == 0x10; } }
```

Comment:
Should be 
  return ((this.Type & 0x10) == 0x10) // check for 0x10 flag
As the type is a set of flags, defined here, where 0x10 is the flag for directory.  Other flags get added on that can change it (e.g. MyMusic is 9238)